_pages/conduct.html

```diff
@@ -123,6 +154,29 @@ <h2>Choose Your Words</h2>
 
         <p>Exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the anti-harassment policy. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.</p>
 
+        <h2>Examples of Inappropriate Behavior</h2>
+        
+        <p>Examples of unacceptable behavior by participants include:</p>
+        <ul>
+          <li>Harassment of any participants in any form</li>
+          <li>Deliberate intimidation, stalking, or following</li>
+          <li>Logging or taking screenshots of online activity for harassment purposes</li>
+          <li>Publishing others’ private information, such as a physical or electronic address, without explicit permission</li>
+          <li>Violent threats or language directed against another person</li>
+          <li>Incitement of violence or harassment towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
+          <li>Creating additional online accounts in order to harass another person or circumvent a ban</li>
+          <li>Sexual language and imagery in online communities or in any conference venue, including talks</li>
+          <li>Insults, put downs, or jokes that are based upon stereotypes, that are exclusionary, or that hold others up for ridicule</li>
+          <li>Excessive swearing</li>
+          <li>Unwelcome sexual attention or advances</li>
+          <li>Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop</li>
+          <li>Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others</li>
+          <li>Sustained disruption of online community discussions, in-person presentations, or other in-person events</li>
+          <li>Continued one-on-one communication after requests to cease</li>
+          <li>Other conduct that is inappropriate for a professional audience including people of many different backgrounds</li>
+        </ul>
+        
+        <p>Community members asked to stop any inappropriate behavior are expected to comply immediately.</p>
```

Comment:
Any change to the markdown file for the non-exhaustive list comment will need to be duplicated here.